src/App.tsx

```diff
@@ -1,27 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { get5First, getAll, getRedGoods } from './api/goods';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import 'bulma/css/bulma.css';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+// стейт - на кожен клік міняємо дані з сервера - передаємо їх як пропси
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const handleClickAll = () => {
+    getAll().then((data) => setGoods(data))
+  }
 
-    <GoodsList goods={[]} />
-  </div>
-);
+  const handleClickMax5 = () => {
+    get5First().then((data) => setGoods(data))
+  }
```

Comment:
you can make it shorter)
  const handleClickMax5 = () => {
    get5First().then(setGoods)
  }